### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+### Bug Fixes
+
+* **account transfer:** remove unimplemented excludeFee option ([cddbbe9](https://github.com/aeternity/aepp-cli-js/commit/cddbbe9f6a063142ac9cb6ed58fd088b18d2aea4))
+* **account transfer:** remove unused denomination option ([4c4d215](https://github.com/aeternity/aepp-cli-js/commit/4c4d21532106f9927f46e58cea9f2afb09190e83))
+* **contract call:** accept arguments as JSON-encoded array ([201a7e0](https://github.com/aeternity/aepp-cli-js/commit/201a7e042d81686eabfbf14e53bcb82b7f9f01c0))
+* **name full-claim:** key of created pointer ([7c278f1](https://github.com/aeternity/aepp-cli-js/commit/7c278f1bfd9724f1a2f8e235fe2ad013c587a9bb))
+* config command ([27a1169](https://github.com/aeternity/aepp-cli-js/commit/27a11697d649e7042bae683cb9a438f63950d9d1))
+* usage show correct command name ([223f05d](https://github.com/aeternity/aepp-cli-js/commit/223f05d819da749894c5de7b640ff37e8cee92bf))
+* **contract:** decode call result, make args flexible ([b0a5cdc](https://github.com/aeternity/aepp-cli-js/commit/b0a5cdc7da289e64121bb17a48ff2e9b55ae8d84))
```
Comment (davidyuk) [inline CHANGELOG.md:36]: For some reason, `refactor` commits are not getting included here, that make changelog quite clear.

Comment (marc0olo) [reply CHANGELOG.md:36]: is sth. important missing now in changelog or do you think we can proceed? not sure if I understand your statement correctly :)

Comment (davidyuk) [reply CHANGELOG.md:36]: I think I saw `refactor` commits in SDK release notes, I just wonder that them are not included here, but this is fine. In general I don't see blockers to release the cli.

**Review by marc0olo — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (marc0olo) [reply CHANGELOG.md:36]: alright, let's go =)

**Pull request merged**